spec/ui/elements.rst

```diff
@@ -0,0 +1,250 @@
+ * **Upgrade** - allows to update the selected cluster.
+ * **Config** - allows to go to the settings of the selected cluster.
+* List of added clusters with information according to table fields.
+* Button to delete an entry from the list of clusters
+* Pagination of the list of clusters
+       
+.. _ui_templates_cluster-services:
+       
+Сluster's service list
+----------------------
+
+The list of cluster services looks like this:
+
+
+Template parts are:
+
+* Navigation Menu with an indication of the decomposition to the selected section, actions that can be performed on the cluster (start, stop, install, update).
+* Cluster name information and link to cluster bundle.
+* Button "Add services" - allows you to add services to the cluster.
+* Left menu with the following sections (active partition is highlighted):
+ * **Main**
+ * **Services**
+ * **Hosts**
+ * **Host - Components**
+ * **Configuration**
+ * **Status**
+ * **Import**
+ * **Actions**
+* Tabular section which includes the following fields:
+ * **Name** - contains information about the name of the service (sorting supported).
+ * **Version** - contains information about the version of the service.
+ * **State** - contains information about the current state of the service (sorting supported).
+ * **Status** - contains information about the current status of the service (sorting supported).
+ * **Actions** - provides a choice of what action to perform on the service (check, expand, restart, shrink, start, stop).
+ * **Import** - ???
+ * **Config** - allows to go to the settings of the selected service.
+* List of added services with information according to table fields.
+* Pagination of the list of clusters
+
+.. _ui_templates_cluster_host_list_view:
+
+Сluster's host list
+--------------------
+
+The list of host include in a cluster based on :ref:`ui_templates_table_view`
+
+Common parameters of template:
+
+* "CLUSTERS" element of "Main Menu" is selected
+* Navigation menu point to a Cluster
+* "Hosts" element of "Left Menu" is selected
+
+:ref:`ui_templates_table_view` columns:
+
+=============== ==============================
+Header          Value
+=============== ==============================
+FQDN            FQDN in string format
+Provider        Link to related HostProvider
+State           State of host
+Action          Action Button
+Config          Link to config of the host
+=============== ==============================
+
+*****
+Forms
+*****
+
+.. _ui_elements_forms_login:
+
+Login
+=====
+
+There are two variants of login form.
+
+First is able to login over Google GAuth2.
+
+.. image:: img/form_login_full.png
+
+
+Second allows to login with user/password only.
+
+.. image:: img/form_login_short.png
+
+Input fields:
+
+* **Login** - is an input to pass a username
+* **Password** - is an input to pass a password. This field is protected and show no password actually.
+
+Buttons:
+
+* **Sign in with Google** - is a button that triggers GAuth2 login process
```

Comment:
This button is displayed only if special conditions are met. By default this button is hidden

Reply:
Fixed